App/VSTS/QuickActions.tsx

```diff
@@ -0,0 +1,81 @@
+/// <reference path="../../office.d.ts" />
+import * as React from 'react';
+import { Provider } from 'react-redux';
+// import { Office } from 'Office';
+import { ItemHyperlink } from  './ItemHyperlink';
+import { FollowButton } from './FollowButton';
+import { ReplyAllButton } from './ReplyAllButton';
+import { CopyButton } from './CopyButton';
+import { FollowStateTypes } from '../statesEZ';
+import { connect } from 'react-redux';
+import { RestButton } from './RestButton';
+import * as ReactDOM from 'react-dom/server';
+
+interface IQuickActionProps {
```

Comment:
Changed - I have passed in the entire workItemState instead of each individual member.